chi/container.py

```diff
@@ -45,18 +45,11 @@
 def create_container(
     name: "str",
     image: "str" = None,
-    image_driver: "str" = DEFAULT_IMAGE_DRIVER,
-    device_profiles: "list[str]" = None,
-    environment: "dict" = None,
```

Comment:
Are we not able to pass environment variables?

Reply:
It's still possible, it will be part of `kwargs` instead of an explicit variable in the function signature